src/main/scala/uk/gov/hmrc/scalatestaccessibilitylinter/AccessibilityLinterMatchers.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.scalatestaccessibilitylinter
+
+import org.scalatest.matchers.{MatchResult, Matcher}
+import play.api.libs.json._
+import scala.collection.immutable.ListMap
+
+trait AccessibilityLinterMatchers {
+  protected val accessibilityLinters: Seq[AccessibilityLinter]
```

Comment:
I can't see how this gets populated? If I try to use this in a test suite I get `value accessibilityLinters in trait AccessibilityLinterMatchers of type Seq[uk.gov.hmrc.scalatestaccessibilitylinter.AccessibilityLinter] is not defined`

Reply:
Works if changed to:

```
  private def process(command: String): ProcessBuilder =
    Process(s"node $command", new File("target/sbtaccessibilitylinter/js"))

  private val accessibilityLinters: Seq[AccessibilityLinter] = Seq(
    new AxeAccessibilityLinter(SystemSubprocess(process("axe")), KnownIssues.loadFromApplicationConfig)
  )
```